tinkerers/S_ANUSHREE.md

```diff
@@ -0,0 +1,11 @@
+## ABOUT ME!!
+*Hi there, myself* **S ANUSHREE..**
+*I am studying Btech in* **INFORMATION TECHNOLOGY** *at* **GOVERNMENT ENGINEERING COLLEGE,PALAKKAD.** *I am 2nd year student.*
+
+*I am currently exploring my skils in different fields and on a quest to find my apt stream.I have currently covered some basics in* **C,Python,Web Designing(HTML/CSS)** *and data structre in C.*
+
+
+## SOCIALS
+**Linkedin:** https://www.linkedin.com/in/s-anushree-923822253
```

Comment:
Heyy Anushree, please use proper markdown syntax to display this URL as a hyperlink.

```
like this 👇
[Linkedin](YOUR_PROFILE_URL)
```